packages/components/src/internal/components/domainproperties/assay/AssayPropertiesPanel.tsx

```diff
@@ -196,6 +197,20 @@ class AssayPropertiesPanelImpl extends React.PureComponent<Props & InjectedDomai
         );
     }
 
+    renderLinkToStudySettings() {
+        const { model } = this.props;
+
+        return (
+            <>
```

Comment:
You don't need this wrapping `<>` here since there is only one element within it (i.e. the div).

Reply:
Oops, thanks for the catch, done!